compat_config.js

```diff
@@ -13,6 +13,7 @@ export default {
       ],
       'plugins': [
         'external-helpers',
+        'babel-plugin-optimize-starts-with',
```

Comment:
I wanted to keep the code as close to `fluent@0.6.0` as possible. We started using `startsWith('-')` in 0.6.0 to tell terms and messages apart. This plugin converts it to `charCodeAt(0) == 45`.